readme.md

```diff
@@ -82,6 +90,15 @@ Content-Length: 112
         "day":"MWF",
         "time":"12-1",
         "location":"KEC"
+        },
+        {
+        "cid": 6,
+        "crn": 11112,
+        "courseName": "CS 122",
+        "instructor": "Mr. Test2",
+        "day":"MWF",
+        "time":"12-1",
+        "location":"KEC"
         }
 ]
 ```
```

Comment:
this comment applies to the http examples that follow. the documentation may be inexact, but it should not be incorrect.